label_studio_ml/examples/watsonx_llm/README.md

```diff
@@ -0,0 +1,100 @@
+
+
+# Integrate WatsonX to Label Studio
```

Comment:
can we add expected project XML labeling configuration to this section?

Reply:
Added below!